backend/src/entity/index.ts

```diff
@@ -1,2 +1,11 @@
 export * from './user.entity';
 export * from './country.entity';
+export * from './vacation.entity';
+export * from './duration_suggestion.entity';
```

Comment:
Use '-' instead of '_' for words separation to follow the nest js file naming convention.